<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parking</class>
 <widget class="QMainWindow" name="Parking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMain">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonCheck">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonEnter">
          <property name="text">
           <string>入库</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonOut">
          <property name="text">
           <string>出库</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonSysInfo">
          <property name="text">
           <string>关于系统</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageCheck">
       <widget class="QToolBox" name="toolBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>761</width>
          <height>391</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>89</height>
          </rect>
         </property>
         <attribute name="label">
          <string>一号区</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetOne">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>89</height>
          </rect>
         </property>
         <attribute name="label">
          <string>二号区</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetTwo">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>89</height>
          </rect>
         </property>
         <attribute name="label">
          <string>三号区</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollAreaThree">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetThree">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>98</width>
           <height>89</height>
          </rect>
         </property>
         <attribute name="label">
          <string>四号区</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_4">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetFour">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSysInfo">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>141</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>价       格：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditPrice">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>110</y>
          <width>210</width>
          <height>50</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>110</y>
          <width>201</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>（元/每小时）</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>200</y>
          <width>141</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>会员折扣：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>200</y>
          <width>210</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>当前状态：系统信息</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>351</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>当前系统计费标准为：5小时以内20元，超过5小时部分每小时10元</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageOut">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>卡        号：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>160</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>类        型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>220</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>入库时间：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>280</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>消费金额：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditCardIdOut">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>100</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxCardType">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>160</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>临时卡</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>会员卡</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lineEditEnterTimeOut">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>220</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditCostOut">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>280</y>
          <width>171</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>280</y>
          <width>41</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>元</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>当前状态：出库</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageEnter">
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>当前状态：入库</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditCardIdEnter">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>100</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>220</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>入库时间：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>卡        号：</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxCardType_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>160</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>临时卡</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>会员卡</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>160</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>类        型：</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSysTime">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>220</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>车  位  号：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditCarPortIdEnter">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>50</y>
          <width>230</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Microsoft PhagsPa</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSure">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>310</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>310</y>
          <width>75</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>车辆管理</string>
    </property>
    <addaction name="actionEnter"/>
    <addaction name="actionOut"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="actionSysInfo"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查询</string>
    </property>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnter"/>
   <addaction name="actionOut"/>
   <addaction name="actionSysInfo"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionEnter">
   <property name="text">
    <string>入库</string>
   </property>
   <property name="toolTip">
    <string>入库</string>
   </property>
  </action>
  <action name="actionOut">
   <property name="text">
    <string>出库</string>
   </property>
   <property name="toolTip">
    <string>出库</string>
   </property>
  </action>
  <action name="actionSysInfo">
   <property name="text">
    <string>系统信息</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出系统</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
